ios/BrightcovePlayer.m

```diff
@@ -158,8 +164,9 @@ - (void)playbackController:(id<BCOVPlaybackController>)controller playbackSessio
     }
 }
 - (void) removeKVOObserver {
```

Comment:
Would suggest calling this `removeKVOObserverIfNeeded` to indicate it will only do so if active.